tests/conftest.py

```diff
@@ -280,6 +280,8 @@
 PUBLIC_DHT_P2P_PUBLIC_KEY_1 = PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_1
 PUBLIC_DHT_P2P_PUBLIC_KEY_2 = PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_2
 
+DEFAULT_LEDGER_LIBP2P_NODE = "fetchai"  # Secp256k1 keys
+
```

Comment:
Does this work if we just change it to eth?

Reply:
nope, now changed to cosmos tho